### ISSUE TITLE
tests should be re-worked to run in parallel

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#127]
It works now. But tests should be re-worked to run in parallel, so that we don't waste time waiting on tests. I will make another issue.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/pull/125#issuecomment-544949486_
